about.html

```diff
@@ -0,0 +1,243 @@
+                    <a href="index.html" class="navLinks">Program</a>
+                </li>
+                <li class="me-5 my-4 d-block text-center">
+                    <a href="index.html" class="navLinks">join</a>
+                </li>
+                <li class="me-5 my-4 d-block text-center">
+                    <a href="index.html" class="navLinks">Sponsor</a>
+                </li>
+                <li class="me-5 my-4 d-block text-center">
+                    <a href="index.html" class="navLinks">News</a>
+                </li>
+              </ul>
+              <div class="d-flex justify-content-center mt-5">
+                <a href="#" class="mobileEnrollButton">Join us now</a>
+              </div>
+          </div>
+      </div>
+</header>
+<article class="aboutHero">
+  <p class="aboutIntroText">It is better a single man
+    than thousands of women</p>
+    <h1 class="text-center text-uppercase">
+      Kenya Women’s<br>summit 2022
+    </h1>
+    <div class="line mt-4 aboutLine"></div>
+    <div class="aboutHeroTexts">
+      <p class="text-center firstParagraph">
+        Kenya Women’s Summit® is one of the largest
+        conferences in kenya and has delivered inspiring programs for 
+        professional and personal empowerment to more than 1,000 women every year.
+      </p>
+      <p class="text-center secondParagraph">
+        Next year we are aiming to grow even further by taking the fight against injustice 
+        and inqeuailty even further by reaching out of our borders to neighbouring countries 
+        like uganda and tanzania.
+      </p>
+      <p class="text-center thirdParagraph">
+        Our MAIN goal is to make every woman gets equality and justice and also
+        to enlighten society about the challenges facing women in the kenyan society 
+      </p>
+    </div>
+      <div class="contactText">
+        <p class="text-center">
+          feel free to reach out to us if you need further clarification about the 
+          work we do for the community <br>
+          <b class="contactEmail">hanningtonepompyy@gmail.com</b>
+        </p>
+      </div>
+</article>
+<article class="womenKenyaLogo">
+  <h2 class="text-center seconday-dark">Kenya Women’s Summit Logo</h2>
+  <div class="line"></div>
+  <div class="logoDescripiton">
+    <p class="text-center desktopDescription">
+      The Kenya women's summit brings together high-achieving women from all over the country. 
+      It gives focus to new ways of thinking and presents aspects we think we already understand from a new and unfamiliar perspective. 
+      This summit enables women to start a dialogue; women who shape their societies and their organisations – women who make their mark.
+      It directs attention to new markets and fields of action for companies and regions, enables the joint exploration of visions and strategies, 
+      and extends the boundaries of previous knowledge. 
+    </p>
```

Comment:
- [ ] Kindly the text here overlaps each other, and maybe it looks good on your screen, but you have also to be sure that your design works correctly on all widths.
- [ ] Kindly you can give the paragraph some margin from left and right to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/168909793-63724db4-16ba-41dd-a396-33c2dbc677a3.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168909875-3654d80d-19a7-4264-98ab-04f59320138d.png)